src/components/20-molecules/input-phone/story.js

```diff
@@ -0,0 +1,34 @@
+import { text, select, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+import changelog from './CHANGELOG.md';
+import readme from './README.md';
+
+export default {
+  title: 'Components',
+  decorators: [withKnobs],
+  parameters: {
+    readme,
+    changelog,
+  },
+};
+
+export const InputPhone = () => {
```

Comment:
pls. add a "controlled react" demo, similair as input text has:
https://axa-ch-webhub-cloud.github.io/plib-feature/feature/input-phone/?path=/story/examples-input-text-react--controlled-uncontrolled

Thanks!!!

Reply:
Hi Marek, you can find it here: https://axa-ch-webhub-cloud.github.io/plib-feature/feature/input-phone/?path=/story/examples-input-phone-react--controlled-uncontrolled